.storybook/main.ts

```diff
@@ -11,6 +11,7 @@ export default {
   features: {
     babelModeV7: true,
   },
+  staticDirs: ['../public'],
```

Comment:
this might work for storybook but I'd be suspicious if this works as a package. You may want to try a build and npm link

Reply:
is gone now, svg is a base64 datauri